spectrocloud/resource_cluster_eks.go

```diff
@@ -804,11 +804,11 @@ func toMachinePoolEks(machinePool interface{}) *models.V1EksMachinePoolConfigEnt
 func toFargateProfileEks(fargateProfile interface{}) *models.V1FargateProfile {
 	m := fargateProfile.(map[string]interface{})
 
-	labels := make([]string, 0)
-	controlPlane, _ := m["control_plane"].(bool)
-	if controlPlane {
-		labels = append(labels, "master")
-	}
+	// labels := make([]string, 0)
```

Comment:
Why do we need to comment this?

Reply:
because it is unused. I will defer to you on whether/how it _should_ be used... or it should be deleted